ROADMAP.md

```diff
@@ -16,7 +16,7 @@ Design: [bit.ly/kfp-v2](https://bit.ly/kfp-v2)
 
 #### Post-v2 Goals (tentative)
 
-* Support local pipeline runner
+* Support local development
```

Comment:
This is the exact language from the slide. The context was to support easy local testing/debugging of primitive components rather than pipelines. There are significant differences between the two. 
/cc @zijianjoy

Reply:
/lgtm
/approve

Reply:
Ah I see, I misunderstood the slide then. To make sure people understand what local development, how do you think about the following description?

```
Support local testing/debugging of primitive components easily.
```